index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nicolas Olaya Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+</head>
+<body>
+    <div class="wrapped">
+        <nav class="navbar">
+            <h2>Welcome</h2>
+            <a href="#" class="options"><i class="las la-bars"></i></a>
+            <a href="#" class="shortcuts">Portfolio</a>
+            <a href="#" class="shortcuts">About</a>
+            <a href="#" class="shortcuts">Contact</a>
+        </nav>
+
+        <main id="homepage">
+            <h1>Hey there. I'm Nicolas<br><span>I'm a software developer</span></h1>
+            
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            <ul class="socials">
+                <li class="social">
+                    <a href="https://github.com/NicolasAndrehh"><i class="lab la-github"></i></a>
+                </li>
+                <li class="social">
+                    <a href="https://www.linkedin.com/in/nicolas-andres-olaya-gamba-3b032b248/"><i class="lab la-linkedin-in"></i></a>
+                </li>
+                <li class="social">
+                    <a href="https://twitter.com/nicolasolaya22"><i class="lab la-twitter"></i></a>
+                </li>
+            </ul>
```

Comment:
Please try to follow the Figma design.
![image](https://user-images.githubusercontent.com/61361037/212153269-9ffb2fc4-76db-467c-b0f4-2de13d7cfd72.png)

This is yours:
![image](https://user-images.githubusercontent.com/61361037/212153388-54a92d22-725f-44c7-ba30-7974cfb8f065.png)